src/loader/tsbgw_interface.h

```diff
@@ -0,0 +1,22 @@
+#ifndef TSBGW_INTERFACE_H
+#define TSBGW_INTERFACE_H
+
+#include <postgres.h>
+#include <fmgr.h>
+
```

Comment:
was chatting with Mat and seems like a good idea to add a file in the versioned extension that exposes these functions within the versioned extension and hides the need to call them via the DirectFunctionCalln interface. Will do that in the next version, esp. as I start working on the scheduler code I think that will make it much easier to read.